Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3
+
+WORKDIR /app
+
+RUN pip install pipenv
+
+# Install dependencies into their own layer so they are cached between code changes
+COPY Pipfile Pipfile.lock ./
+RUN pipenv install --system --deploy
+
+COPY . .
+
+EXPOSE 8080
+
+CMD ["gunicorn", "-c", "gunicorn.py", "wsgi:app"]
```

Comment:
I'm unsure of when we should use something like `dumb-init` or `tini` vs using `gunicorn` directly here. Thoughts?

Reply:
I don’t think there is any need, from my past experience gunicorn seems to handle signals without any trouble. Do you have any docs that indicate we should use `tiny` with gunicorn? Regardless, I think it’s a deployment concern - 

> 
NOTE: If you are using Docker 1.13 or greater, Tini is included in Docker itself. This includes all versions of Docker CE. To enable Tini, just pass the --init flag to docker run.

https://github.com/krallin/tini/blob/master/README.md